config/default.yaml

```diff
@@ -95,8 +95,8 @@ executor:
             # The host or IP of the kubernetes cluster
             host: kubernetes.default
             # The jwt token used for authenticating kubernetes requests
-            # Loaded from /var/run/secrets/kubernetes.io/serviceaccount/token by default
-            # token:
+            # Loaded from /var/run/secrets/kubernetes.io/serviceaccount/token by default if not specified
+            token: 'fakeToken'
```

Comment:
Add this so that it won't break when you run `npm start` locally. `executor-router` will require all executor plugins no matter what default you pick. So when I test it locally using `docker` as my default, it complains that ` /var/run/secrets/kubernetes.io/serviceaccount/token` does not exist which is annoying and does not make any sense.